workloads/centos7.yaml

```diff
@@ -0,0 +1,64 @@
+---
+base_image_url: https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1804_02.qcow2
+base_image_name: Centos 7
+hostname: singlevm
```

Comment:
This field is no longer supported and should be deleted.

Reply:
And by this field I mean 'hostname'

Reply:
@devimc If you just want to remove the hostname field I'll merge this patch this evening.  Everything else looks good.